kortex_driver/launch/kortex_dual_driver.launch

```diff
@@ -56,100 +57,73 @@
     <arg name="right_default_goal_time_tolerance" default="0.5"/> <!--seconds-->
     <arg name="right_default_goal_tolerance" default="0.005"/> <!--radians-->
 
+    <!-- Load arms descriptions -->
+   <param 
+        name="robot_description" 
+        command="$(find xacro)/xacro --inorder $(find kortex_description)/multiple_robots/kortex_dual_robots.xacro 
+            left_arm:=$(arg left_arm) 
+            left_gripper:=$(arg left_gripper)
+            left_dof:=$(arg left_dof)
+            left_vision:=$(arg left_vision)
+            left_sim:=false
+            left_prefix:=$(arg left_prefix)
+            right_arm:=$(arg right_arm)
+            right_gripper:=$(arg right_gripper)
+            right_dof:=$(arg right_dof)
+            right_vision:=$(arg right_vision)
+            right_sim:=false
+            right_prefix:=$(arg right_prefix)"
+    />
+
+    <!-- Start the kortex_driver node -->
```

Comment:
We are in fact starting 2 kortex_driver nodes here, we should modify the comment to illustrate that